backend/sirius-web-services-api/src/main/java/org/eclipse/sirius/web/services/api/images/CustomImage.java

```diff
@@ -0,0 +1,63 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.services.api.images;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import org.eclipse.sirius.web.annotations.Immutable;
+import org.eclipse.sirius.web.annotations.graphql.GraphQLField;
+import org.eclipse.sirius.web.annotations.graphql.GraphQLID;
+import org.eclipse.sirius.web.annotations.graphql.GraphQLNonNull;
+import org.eclipse.sirius.web.annotations.graphql.GraphQLObjectType;
+
+/**
+ * Lightweight DTO representing a custom image (without the content).
+ *
+ * @author pcdavid
+ */
+@Immutable
+@GraphQLObjectType
```

Comment:
Does this need to appear in the GraphQL API? Why?

Reply:
Careless copy/paste. I'll remove it.